guides/v2.1/install-gde/install/cli/install-cli-uninstall.md

```diff
@@ -5,81 +5,73 @@ title: Uninstall or reinstall Magento
 menu_title: Uninstall or reinstall Magento
 menu_node:
 menu_order: 5
-redirect_from:
-  - /guides/v2.0/install-gde/install/install-cli-uninstall.html
 functional_areas:
   - Install
   - System
   - Setup
 ---
 
 ## First steps {#instgde-cli-before}
+
 {% include install/first-steps-cli.md %}
-In addition to the command arguments discussed here, see [Common arguments]({{ page.baseurl }}/install-gde/install/cli/install-cli-subcommands.html#instgde-cli-subcommands-common).
+In addition to the command arguments discussed here, see [Common arguments][].
```

Comment:
To my knowledge, you don't need to add the extra brackets after the links. Should just be [Common arguments] without the extra square brackets that follow. Goes for all of your links.

Reply:
I was instructed otherwise, so I will get the straight skinny on it.